fish-rust/src/builtins/math.rs

```diff
@@ -189,24 +187,26 @@ fn evaluate_expression(
 
             streams
                 .err
-                .append(sprintf!("%ls: Error: %ls\n"L, cmd, error_message));
-            streams.err.append(sprintf!("'%ls'\n"L, expression));
+                .append(sprintf!(L!("%ls: Error: %ls\n"), cmd, error_message));
+            streams.err.append(sprintf!(L!("'%ls'\n"), expression));
```

Comment:
while we are at I think we should remove it from printf format strings; those magically work with plain literals. Same elsewhere.